tensorboard/plugins/core/core_plugin.py

```diff
@@ -50,18 +51,24 @@ class CorePlugin(base_plugin.TBPlugin):
 
     plugin_name = "core"
 
-    def __init__(self, context):
+    def __init__(self, context, include_debug_info=None):
         """Instantiates CorePlugin.
 
         Args:
           context: A base_plugin.TBContext instance.
+          include_debug_info: If true, `/data/environment` will include some
+            basic information like the TensorBoard server version. Disabled by
+            default to prevent surprising information leaks in custom builds of
+            TensorBoard.
         """
+        self._flags = context.flags
```

Comment:
Apropos of #2801 and comments on #4411: I don’t mind this because it
really does want to get all of `flags` rather than just using it as a
shortcut for some specific properties, and also because this
functionality really could be part of the _actual_ core, anyway.
But open to pushback if you disagree.

Reply:
Yeah, I agree it's fine for now, although if you don't mind editing #2801 after this goes in so it reflects the current state that'd be helpful.